helix-core/src/test/java/org/apache/helix/manager/zk/TestParticipantManager.java

```diff
@@ -0,0 +1,183 @@
+package org.apache.helix.manager.zk;
+
```

Comment:
Thanks for adding the test!

Reply:
Sure. I should have added the test in the prior PR https://github.com/apache/helix/pull/642